src/types/Venue.ts

```diff
@@ -22,25 +24,7 @@ export interface Venue {
   template: VenueTemplate;
   name: string;
   entrance?: EntranceStepConfig[];
-  config?: {
-    theme: {
-      primaryColor: string;
-      backgroundColor?: string;
-    };
-    landingPageConfig: {
-      coverImageUrl: string;
-      subtitle: string;
-      description?: string;
-      presentation: string[];
-      bannerImageUrl?: string;
-      checkList: string[];
-      iframeUrl?: string;
-      joinButtonText?: string;
-      quotations?: Quotation[];
-    };
-    memberEmails?: string[];
-    showRangers?: boolean;
-  };
+  config?: VenueConfig;
```

Comment:
We have keys in the root of `Venue`, and the `config` key, do we want to keep this in config?

Reply:
@thecadams said it makes sense to use `config` here, we can do a v2 of `Venues` type at some stage and cleanup some of the other keys if it makes sense to. But canonically, use `config`

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/793/commits/d38a1bcc34f80038394d3705cf3e917120abc1f4